packages/lesswrong/components/tagging/FooterTagList.tsx

```diff
@@ -321,16 +341,19 @@ const FooterTagList = ({
               smallText={smallText}
               highlightAsAutoApplied={highlightAutoApplied && tagRel?.autoApplied}
               link={link}
+              noBackground={noBackground}
+              neverCoreStyling={neverCoreStyling}
             />
           )
       )}
       {!hidePostTypeTag && postType}
+      {eventTag}
       {annualReviewMarketInfo && highlightMarket(annualReviewMarketInfo) && (
         <PostsAnnualReviewMarketTag post={post} annualReviewMarketInfo={annualReviewMarketInfo} />
       )}
       {currentUser && !hideAddTag && (
         <AddTagButton onTagSelected={onTagSelected} isVotingContext tooltipPlacement={tooltipPlacement}>
-          {useAltAddTagButton && <span className={classes.altAddTagButton}>+</span>}
+          {useAltAddTagButton && <span className={classNames(classes.altAddTagButton, {[classes.noBackground]: noBackground})}>+</span>}
```

Comment:
```suggestion
          {useAltAddTagButton && <span className={classNames(classes.altAddTagButton, noBackground && classes.noBackground)}>+</span>}
```

I believe this is the preferred method

Reply:
why is that preferred? I thought we'd been consistently doing it the other way.

Reply:
https://github.com/ForumMagnum/ForumMagnum/pull/9118

Reply:
TIL